apps/ehr/tests/e2e/specs/in-person/inHouseLabsPage.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { Page, test } from '@playwright/test';
+import { DateTime } from 'luxon';
+import { CssHeader } from 'tests/e2e/page/CssHeader';
+import { FinalResultPage } from 'tests/e2e/page/FinalResultPage';
+import { openInPersonProgressNotePage } from 'tests/e2e/page/in-person/InPersonProgressNotePage';
+import { PerformTestPage } from 'tests/e2e/page/PerformTestPage';
+import { SideMenu } from 'tests/e2e/page/SideMenu';
+import { ResourceHandler } from '../../../e2e-utils/resource-handler';
+import { expectAssessmentPage } from '../../page/in-person/InPersonAssessmentPage';
+import { expectOrderDetailsPage, OrderInHouseLabPage } from '../../page/OrderInHouseLabPage';
+
+const PROCESS_ID = `inHouseLabsPage.spec.ts-${DateTime.now().toMillis()}`;
+const resourceHandler = new ResourceHandler(PROCESS_ID, 'in-person');
+
+const TEST_TYPE = 'Flu A';
+const CPT_CODE = '87501';
+const DIAGNOSIS = 'Situs inversus';
+const SOURCE = 'Nasopharyngeal swab';
+const STATUS_ORDERED = 'ORDERED';
+const STATUS_COLLECTED = 'COLLECTED';
+const STATUS_FINAL = 'FINAL';
+const TEST_RESULT_DETECTED = 'Detected';
+const SECTION_TITLE = 'In-House Labs';
+
+test.beforeEach(async () => {
+  if (process.env.INTEGRATION_TEST === 'true') {
+    await resourceHandler.setResourcesFast();
+  } else {
+    await resourceHandler.setResources();
+    await resourceHandler.waitTillAppointmentPreprocessed(resourceHandler.appointment.id!);
+  }
+});
+
+test.afterEach(async () => {
+  await resourceHandler.cleanupResources();
+});
+
+test('IHL-1 In-house labs. Happy Path - Order In-house Lab, Collect Sample, Perform test, Final view, Progress Note ', async ({
```

Comment:
can we split up this test into smaller subtests, even if they end up being in series not parallel? we'd be able to see which ones fail.

i think for this one a good split would be based on which page you're testing, eg `orderInHouseLabPage` or `performTestPage`